Lib/test/test_sys_settrace.py

```diff
@@ -257,6 +266,10 @@ def tearDown(self):
         if self.using_gc:
             gc.enable()
 
+    @staticmethod
+    def make_tracer():
```

Comment:
add a comment mentioning why is this needed at all. (letting subclasses such as XYZ override it)